packages/client/src/pages/admin/pages/RegEdit.tsx

```diff
@@ -259,6 +261,9 @@ export default class RegEdit extends Page<PageProps, RegEditState> {
 					<Label>Website name</Label>
 					<TextInput name="Name" />
 
+					<Label>Unit name</Label>
```

Comment:
Can you be more specific? What's the difference between 'Unit name' and 'Website name'? Can we pull 'Unit name' from CAPWATCH data?

Reply:
'Unit Name' in CAPWATCH data is sometimes ST MARYS COMP SQDN where this entry allows for 'St. Mary's Composite Squadron' to be placed on report headers.

Reply:
Ok. Can we make the label more specific, or provide documentation somewhere about what the difference between 'Unit' and 'Website' names are?

```suggestion
					<Label>Full unit name (used for reports)</Label>
```